lib/config.js

```diff
@@ -80,10 +80,15 @@ const conf = convict({
     }
   },
   expiration: {
-    code: {
+    accessTokenS: {
```

Comment:
nit: `expiration.accessTokenInSeconds`? is this format used anywhere else? (adding S and Ms)?